### ISSUE TITLE
app fails to launch

### ISSUE THREAD

**Issue by ghost** [itchio/itch#2911]
running Pop!_OS 22.10, getting similar results to everyone else.

running ~/.itch/itch outputs
```
2023/02/27 00:41:46 itch-setup will log to /tmp/itch-setup-log.txt
2023/02/27 00:41:46 =========================================
2023/02/27 00:41:46 itch-setup "v1.26.0, built on Apr 21 2021 @ 21:48:12, ref 48f97b3e7b0b065a2478811b8d0ebcae414845fd" starting up at "2023-02-27 00:41:46.894871994 -0500 EST m=+0.001934571" with arguments:
2023/02/27 00:41:46 "/home/protodrew/.itch/itch-setup"
2023/02/27 00:41:46 "--prefer-launch"
2023/02/27 00:41:46 "--appname"
2023/02/27 00:41:46 "itch"
2023/02/27 00:41:46 "--"
2023/02/27 00:41:46 =========================================
2023/02/27 00:41:46 App name specified on command-line: itch
2023/02/27 00:41:46 Locale:  en-US
2023/02/27 00:41:46 Initializing installer GUI...
2023/02/27 00:41:46 Using GTK UI
2023/02/27 00:41:46 Initializing (itch) multiverse @ (/home/protodrew/.itch)
2023/02/27 00:41:46 (/home/protodrew/.itch)(current = "25.5.1", ready = "")
2023/02/27 00:41:46 Launch preferred, attempting...
2023/02/27 00:41:46 Launching (25.5.1) from (/home/protodrew/.itch/app-25.5.1)
2023/02/27 00:41:46 Kernel should support SUID sandboxing, leaving it enabled
2023/02/27 00:41:46 App launched, getting out of the way
```

running ~/.itch/app-25.5.1/itch outputs
```
05:39:24.142 INFO itch@25.5.1 on electron@10.4.5 in production
05:39:24.143 DEBUG (m/reac/preb/load-preferences) imported preferences: {"downloadSelfUpdates":true,"offlineMode":false,"installLocations":{},"defaultInstallLocation":"appdata","isolateApps":false,"closeToTray":true,"readyNotification":true,"showAdvanced":false,"openAtLogin":false,"openAsHidden":false,"manualGameUpdates":false,"preventDisplaySleep":true,"preferOptimizedPatches":false,"disableBrowser":false,"enableTabs":false}
05:39:24.245 DEBUG (m/reac/preb/load-preferences) imported preferences: {"downloadSelfUpdates":true,"offlineMode":false,"installLocations":{},"defaultInstallLocation":"appdata","isolateApps":false,"closeToTray":true,"readyNotification":true,"showAdvanced":false,"openAtLogin":false,"openAsHidden":false,"manualGameUpdates":false,"preventDisplaySleep":true,"preferOptimizedPatches":false,"disableBrowser":false,"enableTabs":false}
05:39:24.249 DEBUG (m/net/register-itch-protocol) Registering itch: protocol for session with user agent Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) itch/25.5.1 Chrome/85.0.4183.121 Electron/10.4.5 Safari/537.36
05:39:24.274 INFO (m/reac/preboot) preboot ran in 29.00ms
05:39:24.275 DEBUG (m/reac/open-at-login) Updating login item settings, open=false, hidden=false
05:39:24.275 DEBUG (m/reac/open-at-login) Copying /usr/share/applications/io.itch.itch.desktop => /home/protodrew/.config/autostart/io.itch.itch.desktop
05:39:24.275 DEBUG (m/reac/winds) Waiting on root window to dispatch boot
[43153:0227/003925.068727:FATAL:gpu_data_manager_impl_private.cc(442)] GPU process isn't usable. Goodbye.
fish: Job 1, '~/.itch/app-25.5.1/itch' terminated by signal SIGTRAP (Trace or breakpoint trap)
```

both result in a gtk window popping up (correctly styled to my gtk theme if that helps), and almost immediately closing, with either a clean exit or a sigtrap

**Comment by leafo** [comment]
Closing out old tickets for very old versions of the app. Please update to 26.9.0 or above and create a new issue if you are still having problems.
